PowerSimulationsPlots/src/fuel_results.jl

```diff
@@ -0,0 +1,211 @@
+order = ([
+    "Nuclear",
+    "Coal",
+    "Hydro",
+    "Gas_CC",
+    "Gas_CT",
+    "Storage",
+    "Oil_ST",
+    "Oil_CT",
+    "Sync_Cond",
+    "Wind",
+    "Solar",
+    "CSP",
+    "curtailment",
```

Comment:
Why is `"curtailment"` alone lower case `c`?

Reply:
This was moved to other repo